app/ts/background/background.ts

```diff
@@ -102,6 +105,7 @@ export const simulateGovernanceContractExecution = async (pendingTransaction: Pe
 		}
 		return await visualizeSimulatorState(governanceContractSimulationState, ethereum)
 	} catch(error) {
+		console.warn(error)
```

Comment:
Debugging code, or intentionally left in?  Since this isn't the terminal handler of errors, my instinct is to not log it to the console.  I generally reserve that for errors that are unhandled anywhere else (the last point of handling).

Reply:
If something goes wrong, I feel its good that there's some place where you can read more details on what went wrong, even thought its expected that errors can occur. I am not sure how else handle that